### PR TITLE
Add Latest Version Badge

### LINKED ISSUE #2445: Add target/current revision to status badge
It would be very useful to have the target version displayed in the status-badge. That way you know relative to what the application is (un)synced and (un)healthy.

Current info: {health} | {sync status}
Proposal: {health} | {sync status} | {target revision}

It might also be worth having a separate badge containing both current revision and target revision.

### PR DESCRIPTION
**Pull request by codenio**
added latest version badge for agro-cd

fixes: #2445

### CODE REVIEW

**Review by alexec — APPROVED** [review]

**Review by alexec — COMMENTED** [review]
I've questions. How does this badge get updated when a release is made please?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![slack](https://img.shields.io/badge/slack-argoproj-brightgreen.svg?logo=slack)](https://argoproj.github.io/community/join-slack)
 [![codecov](https://codecov.io/gh/argoproj/argo-cd/branch/master/graph/badge.svg)](https://codecov.io/gh/argoproj/argo-cd)
+[![Latest Version](https://img.shields.io/badge/agro--cd-v1.2.3-blue)](https://github.com/argoproj/argo-cd/releases/tag/v1.2.3)
```
Comment (jannfis) [inline README.md]: Can we name this `argo` instead of `agro` please?

Comment (codenio) [reply README.md]: Sure, sry for the typo

**Comment by codenio** [comment]
> I think this is a static badge for the README of ArgoCD's GitHub page and does not fix #2445

Yes it is a static badge, which demands the maintainer to update during a release. 
As it was not nentioned to be automated in propsed issue. I felt a static one would resolve it.

**Pull request merged**